service/controller/v10/templates/cloudformation/guest/security_groups.go

```diff
@@ -14,6 +14,13 @@ const SecurityGroups = `{{define "security_groups" }}
         ToPort: {{ .Port }}
         CidrIp: {{ .SourceCIDR }}
       {{ end }}
+      {{- if .APIWhitelistEnabled }}
+      -
+        IpProtocol: tcp
+        FromPort: 0
+        ToPort: 443
+        CidrIp: !Join [ "/", [ !Ref NATEIP, "32" ] ]
```

Comment:
Good that you can use the ref to get the EIP. I thought you needed it in the operator rather than the CF templates. Much better not to use the EC2 API.

Reply:
Yeah, this saved me tons of work and race conditions :)